### PR TITLE
Support dell rpl products

### PR DESCRIPTION
**Pull request by gongjun-song**

### CODE REVIEW

**Review by bardliao — DISMISSED** [review]

File: sound/soc/intel/boards/sof_sdw.c [hunk 1]
```diff
@@ -356,6 +356,16 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 					RT711_JD2 |
 					SOF_SDW_FOUR_SPK),
 	},
+	{
```
Comment (plbossart) [inline sound/soc/intel/boards/sof_sdw.c]: please add a RPL separator between platforms, as we've done for TGL, ADL, SDCA before.

**Review by plbossart — CHANGES_REQUESTED** [review]
nit-pick below @gongjun-song 

I also think we are missing some platforms, no? there's only 2 skus here.

**Comment by gongjun-song** [comment]
> nit-pick below @gongjun-song
> 
> I also think we are missing some platforms, no? there's only 2 skus here.

@plbossart Yes, Dell has three platforms, but the 0C11 is still being debugged.
@mengdonglin recommends submitting PR first for two platforms that are already working well.

File: sound/soc/intel/boards/sof_sdw.c (hunk 1 above)
Comment (gongjun-song) [reply sound/soc/intel/boards/sof_sdw.c]: Thanks, will update

**Review by plbossart — APPROVED** [review]

**Review by kv2019i — APPROVED** [review]
Looks good, thanks @gongjun-song !

**Review by bardliao — APPROVED** [review]

**Review by ujfalusi — APPROVED** [review]

**Comment by plbossart** [comment]
SOFCI TEST

**Pull request merged**